### ISSUE TITLE
KONKE switch use int value instead of bool

### ISSUE THREAD

**Issue by doudz** [doudz/zigate#144]
https://github.com/doudz/homeassistant-zigate/issues/66

**Comment by doudz** [comment]
```
 {
            "addr": "d067",
            "discovery": "auto-discovered",
            "endpoints": [
                {
                    "clusters": [
                        {
                            "attributes": [
                                {
                                    "attribute": 4,
                                    "data": "Konke",
                                    "name": "manufacturer",
                                    "value": "Konke"
                                },
                                {
                                    "attribute": 5,
                                    "data": "3AFE170100510001",
                                    "name": "type",
                                    "type": "str",
                                    "value": "3AFE170100510001"
                                },
                                {
                                    "attribute": 0,
                                    "name": "zcl_version",
                                    "value": null
                                },
                                {
                                    "attribute": 1,
                                    "name": "application_version",
                                    "value": null
                                },
                                {
                                    "attribute": 2,
                                    "name": "stack_version",
                                    "value": null
                                },
                                {
                                    "attribute": 3,
                                    "name": "hardware_version",
                                    "value": null
                                },
                                {
                                    "attribute": 6,
                                    "name": "datecode",
                                    "value": null
                                },
                                {
                                    "attribute": 7,
                                    "name": "power_source",
                                    "value": null
                                },
                                {
                                    "attribute": 8
                                },
                                {
                                    "attribute": 9
                                },
                                {
                                    "attribute": 10
                                },
                                {
                                    "attribute": 11
                                },
                                {
                                    "attribute": 65533
                                }
                            ],
                            "cluster": 0
                        },
                        {
                            "attributes": [
                                {
                                    "attribute": 32,
                                    "data": 29,
                                    "name": "battery_voltage",
                                    "type": "float",
                                    "value": 2.9
                                }
                            ],
                            "cluster": 1
                        },
                        {
                            "attributes": [
                                {
                                    "attribute": 0,
                                    "data": true,
                                    "name": "onoff",
                                    "type": "bool",
                                    "value": true
                                }
                            ],
                            "cluster": 6
                        }
                    ],
                    "device": 2,
                    "endpoint": 1,
                    "in_clusters": [
                        0,
                        1,
                        3,
                        4,
                        5,
                        6
                    ],
                    "out_clusters": [
                        3
                    ],
                    "profile": 260
                }
            ],
            "generictype": "",
            "info": {
                "addr": "d067",
                "id": 0,
                "ieee": "086bd7fffebfe2d9",
                "last_seen": "2019-09-04 11:12:33",
                "lqi": 144,
                "mac_capability": "10000000",
                "power_type": 0
            }
        },
```

**Comment by doudz** [comment]
fixed in 0.34
